examples/all-clusters-app/telink/include/AppTask.h

```diff
@@ -70,6 +79,11 @@ class AppTask
 
     static AppTask sAppTask;
     PWMDevice mPwmIdentifyLed;
+
+#if CONFIG_CHIP_FACTORY_DATA
+    // chip::DeviceLayer::FactoryDataProvider<chip::DeviceLayer::InternalFlashFactoryData> mFactoryDataProvider;
```

Comment:
Do not check in commented-out code.
If you want to also menction `InternalFlashFactoryData` exists, then comment stating that, don't put a line of code without context.

Reply:
Fixed